BrightID/locales/en/translation.json

```diff
@@ -38,6 +38,10 @@
     "setupTrustedConnections": {
       "description": "Setup your recovery connections to enable social account recovery",
       "title": "Setup social recovery"
+    },
+    "bituVerification": {
+      "description": "More details",
```

Comment:
I think this should be a one-sentence description, similar to the other items.

Reply:
but I think having "More details" as text make this more clear for users that they should press this. I'm worried that using a text like other achievements while other achievements has no independent screen, may result in users do not try pressing that.